Cargo.toml

```diff
@@ -16,6 +16,7 @@ futures = "0.3"
 libc = "0.2.79"
 vsock = "0.2.4"
 tokio = { version = "1", features = ["net"] }
+tonic = "0.6.2"
```

Comment:
This is quite a big dependency to add not part of the core functionality of this crate. Can you make it optional, enabled by a feature?

Reply:
I added in the feature gate. I could be convinced to a different feature name pretty easy, but I used `tonic-conn`.